From 9a6ff8cafa97717560f9eec72b4badcf31db89f7 Mon Sep 17 00:00:00 2001
From: Jaehoon Chung <jh80.chung@samsung.com>
Date: Fri, 15 Jul 2016 10:54:51 +0900
Subject: [PATCH 26/95] mmc: dw_mmc: rockchip: unset the MMC_CAP_ERASE flag

Git-commit: 2c4c7146c7f6cffcbd71f1df95e6bc6727324e05
Patch-mainline: v4.8-rc1
References: bsc#1033484

In dw_mmc.c, it's enabled by default.
It doesn't need to set MMC_CAP_ERASE in rockchip anymore.

Signed-off-by: Jaehoon Chung <jh80.chung@samsung.com>
Acked-by: Shawn Lin <shawn.lin@rock-chips.com>
Signed-off-by: Ulf Hansson <ulf.hansson@linaro.org>
Signed-off-by: Mian Yousaf Kaukab <yousaf.kaukab@suse.com>
---
 drivers/mmc/host/dw_mmc-rockchip.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/mmc/host/dw_mmc-rockchip.c b/drivers/mmc/host/dw_mmc-rockchip.c
index 4c41f94..4864e66 100644
--- a/drivers/mmc/host/dw_mmc-rockchip.c
+++ b/drivers/mmc/host/dw_mmc-rockchip.c
@@ -295,10 +295,10 @@ static int dw_mci_rockchip_init(struct dw_mci *host)
 
 /* Common capabilities of RK3288 SoC */
 static unsigned long dw_mci_rk3288_dwmmc_caps[4] = {
-	MMC_CAP_RUNTIME_RESUME | MMC_CAP_ERASE | MMC_CAP_CMD23, /* emmc */
-	MMC_CAP_RUNTIME_RESUME | MMC_CAP_ERASE | MMC_CAP_CMD23, /* sdmmc */
-	MMC_CAP_RUNTIME_RESUME | MMC_CAP_ERASE | MMC_CAP_CMD23, /* sdio0 */
-	MMC_CAP_RUNTIME_RESUME | MMC_CAP_ERASE | MMC_CAP_CMD23, /* sdio1 */
+	MMC_CAP_RUNTIME_RESUME | MMC_CAP_CMD23, /* emmc */
+	MMC_CAP_RUNTIME_RESUME | MMC_CAP_CMD23, /* sdmmc */
+	MMC_CAP_RUNTIME_RESUME | MMC_CAP_CMD23, /* sdio0 */
+	MMC_CAP_RUNTIME_RESUME | MMC_CAP_CMD23, /* sdio1 */
 };
 static const struct dw_mci_drv_data rk2928_drv_data = {
 	.init			= dw_mci_rockchip_init,
-- 
2.11.0

